wye/base/management/commands/sample_data.py

```diff
@@ -96,7 +117,344 @@ def create_organisations(self, counter=None):
+                'workshop_level': WorkshopLevel.BEGINNER,
+                'workshop_section': python2,
+                'status': WorkshopStatus.COMPLETED
+            },
+            {
+                'participants': 100,
+                'expected_date': date(2015, 1, 11),
+                'description': 'test',
+                'requester': organisation,
+                'location': location,
+                'workshop_level': WorkshopLevel.BEGINNER,
+                'workshop_section': python2,
+                'status': WorkshopStatus.COMPLETED
+            },
+            {
+                'participants': 100,
+                'expected_date': date(2015, 1, 12),
+                'description': 'test',
+                'requester': organisation,
+                'location': location,
+                'workshop_level': WorkshopLevel.BEGINNER,
+                'workshop_section': python2,
+                'status': WorkshopStatus.COMPLETED
+            },
+            {
+                'participants': 100,
+                'expected_date': date(2015, 6, 12),
+                'description': 'test',
+                'requester': organisation,
+                'location': location,
+                'workshop_level': WorkshopLevel.BEGINNER,
+                'workshop_section': python2,
+                'status': WorkshopStatus.COMPLETED
+            },
+            {
+                'participants': 100,
+                'expected_date': date(2015, 11, 12),
+                'description': 'test',
+                'requester': organisation,
+                'location': location,
+                'workshop_level': WorkshopLevel.BEGINNER,
+                'workshop_section': python2,
+                'status': WorkshopStatus.COMPLETED
+            },
+            {
+                'participants': 100,
+                'expected_date': date(2015, 11, 13),
+                'description': 'test',
+                'requester': organisation,
+                'location': location,
+                'workshop_level': WorkshopLevel.BEGINNER,
+                'workshop_section': python2,
+                'status': WorkshopStatus.COMPLETED
+            },
+            {
+                'participants': 100,
+                'expected_date': date(2015, 11, 14),
+                'description': 'test',
+                'requester': organisation,
+                'location': location,
+                'workshop_level': WorkshopLevel.BEGINNER,
+                'workshop_section': python2,
+                'status': WorkshopStatus.COMPLETED
+            },
+            {
+                'participants': 100,
+                'expected_date': date(2015, 11, 15),
+                'description': 'test',
+                'requester': organisation,
+                'location': location,
+                'workshop_level': WorkshopLevel.BEGINNER,
+                'workshop_section': python2,
+                'status': WorkshopStatus.COMPLETED
+            }
+        ]
```

Comment:
How about using `faker` and/or `random.choices` to created random data inside the loop itself. Keep it DRY!